package.json

```diff
@@ -126,6 +126,7 @@
 		"phpegjs": "1.0.0-beta7",
 		"postcss": "7.0.13",
 		"progress": "2.0.3",
+		"puppeteer": "1.19.0",
```

Comment:
It's a bit confusing to me what the requirements are for dependencies to be part of this list. E.g. I see `jest` as a peer dependency of several packages, but it doesn't appear in this list.

Reply:
It shouldn't be here. It's referenced in `@wordpress/scripts`.

I'm sure this list has some legacy stuff which should no longer be here. The general rule is that all dev dependencies used in WordPress packages should be listed here as we manage them globally. We use `@wordpress/scripts` to run Puppeteer driven tests so both `jest` and `puppeteer` are already set as dependencies of `@wordpress/scripts`.

Reply:
I fixed it in bb406e6.